cardano-node/src/Cardano/Common/Parsers.hs

```diff
@@ -249,6 +252,15 @@ parseValidateDB =
       <> help "Validate all on-disk database files"
     )
 
+parseShutdownIPC :: Parser (Maybe Fd)
```

Comment:
I don't think this will compile on Windows because it's `HANDLE` there. Perhaps use `Int` here and sort out the rest when it's time to use fdToHandle.

Reply:
GHC uses the MS C lib on Windows which does actually handle these Fds.